code/style.css

```diff
@@ -1,7 +1,309 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+body {
+  min-height: 100vh;
+  background:linear-gradient(#F3C5C5, #C1A3A3, #886F6F, #694E4E);
+  font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
```

Comment:
I'm curious, I've always just seen two different fonts. Why four? Is there a reason to that?

Reply:
The one you see here is 'Impact', the other ones are fallback-fonts (in case one browser isn't able to show the Impact-font it's going to try out the font that comes next.)